relationships.md

```diff
@@ -0,0 +1,116 @@
+# Relationship Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint contains the actual relationships connecting 2 items.
+It uses a [relationship type](relationshiptypes.md) and the 2 items with HAL links
+
+## Main Endpoint
+**/api/core/relationships**
+
+A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes
+
+## Single Relationship
+**/api/core/relationships/<:id>**
+
+A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationships/530
+The leftId and rightId parameter in JSON may still be present, but they will be removed soon (requires other PRs to be merged first)
+
+```json
+{
+  "id": 530,
+  "relationshipTypeId": 0,
+  "leftPlace": 1,
+  "rightPlace": 1,
+  "type": "relationship",
+  "_links": {
+    "relationshipType": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/1"
+    },
+    "self": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/relationships/530"
+    },
+    "leftItem": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/items/e98b0f27-5c19-49a0-960d-eb6ad5287067"
+    },
+    "rightItem": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/items/0ffbee3f-e7ea-42bc-92fe-2fbef1a52c0f"
+    }
+  },
+  "_embedded": {
+    "relationshipType": {
+      "id": 1,
+      "leftLabel": "isAuthorOfPublication",
+      "rightLabel": "isPublicationOfAuthor",
+      "leftMinCardinality": 5,
```

Comment:
This comment is not really "solvable" in this PR as it belongs more to the general approach adopted by the configurable entities prototype. Support min cardinality on bidirectional relationships is complex: how is possible to guarantee its respect when both are > 0? say that left and right cardinality are both 1, how can I create two items linked together by the previous defined relationship without violate such constraint? when the constraint is expected to be checked?

About this PR I suggest to pick a concrete meaningful example for the leftMinCardinality the 5 here is just random (maybe journal - journalVolume can be set with leftMinCardinality: 1)

Reply:
The sample values here were from older tests and will be updated in the contracted. The plan is not to include a full CRIS relationship in the samples, but the samples will be updated to be closer to real-life use